src/server/auth.ts

```diff
@@ -53,3 +54,43 @@ export default async function initializePatronTokenAuth(req: NextRequest) {
 
   return patronTokenResponse
 }
+
+/**
+ * Creates and returns redirect url from initializePatronTokenAuth() response. Call this function only with an un-authenticated patron,
+ * i.e., patronTokenResponse.isTokenValid must be false.
+ */
+export function getLoginRedirect(req) {
+  const protocol = "http"
```

Comment:
This is fine for testing locally but it should be https since we require the secure protocol on qa/prod. Is it available in the `req` object?

Reply:
`req.headers["x-forwarded-proto"]` or `req.protocol` are available but they're always undefined locally so I'm unsure. I think there's a broader issue with testing this because the environment is not consistent (setting APP_ENV doesn't seem to affect it)

Reply:
I assume `req.protocol` would be defined on prod and it would be `https`

Reply:
so do we want protocol to be req.protocol if it _is_ defined?